app/javascript/src/components/editor/LineFinder.svelte

```diff
@@ -63,8 +68,11 @@
       // Attempt to find line marker starting at current line moving up
       let linemarkerStart = -1
       let i = intValue
-      while (linemarkerStart == -1 && i) {
+      while (i) {
         linemarkerStart = splitCompiled[i].indexOf("[linemarker]")
+        if (linemarkerStart != -1) {
+          break
+        }
```

Comment:
I personally prefer putting things on one line when it still reads well enough

```suggestion
        if (linemarkerStart != -1) break
```